scripts/autobuild.sh

```
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+set -e
+
+script_dir="$( cd "$( dirname "$0"  )" && pwd  )"
+project_dir=$(dirname "$script_dir")
+systype=$(uname -s)
```

Shellcheck (0.10.0)

[warning] systype appears unused. Verify use (or export if used externally).

(SC2034)